app.js

```diff
@@ -0,0 +1,81 @@
+require('dotenv').config();
+const Discord = require('discord.js');
+const client = new Discord.Client();
+
+client.once('ready', () => {
+    console.log('Ready');
+});
+
+client.login(process.env.BOT_TOKEN);
+
+// Use exclamation mark as the default prefix
+const prefix = process.env.PREFIX || '!';
+
+const help = '**Bot commands**\n✨ Start pairings with `!start`\n🛑 Stop pairings with `!stop`\n💖 Set your pronouns with `!pronouns PRONOUNS`\n 💻 Share your GitHub profile with `!github GITHUB_URL`\n💼 Share your LinkedIn profile with `!linkedin LINKEDIN_URL`\n🐙 Share a fun fact about yourself with `!funfact FUN_FACT`\n❔ Show this menu with `!help`';
+
+client.on('message', message => {
+    // Ignore other bot messages
+    if (message.author.bot) return;
+
+    // Remove irc username suffix
+    const messageContent = message.content.replace(/<.*> /, '');
+
+    // Ignore messages in public places (for now)
+    if (message.channel.type !== 'dm') return;
+
+    let args = messageContent
+        .slice(prefix.length)
+        .trim()
+        .split(/ +/g);
+    const command = args.shift().toLowerCase();
+    args = args.join(' ');
+
+    let response = null;
+
+    switch(command) {
+    case 'start':
```

Comment:
Perhaps we could replace this with a map from an object to command? what do you think?
```
// this gives the start message
COMMANDS['start'] 
```
Then we can even make it a bit more powerful by mapping from command to a function name then we could pass things into the function such as the user's name, to give the bot a more personal feel.

Reply:
As discussed in the chat – this is an awesome idea to make the command responses neatly organized in one place. As things progress we may not be able to use a 1-1 mapping with the user commands and the responses, as there could be multiple responses for a single command. I think we'll be able to expand on this in the next iterations. @smartclash also suggested to use https://github.com/discordjs/Commando